embed/config.go

```diff
@@ -315,6 +315,9 @@ type Config struct {
 	loggerCore        zapcore.Core
 	loggerWriteSyncer zapcore.WriteSyncer
 
+	// If EnableGRPCGateway is false, grpc gateway will be disabled.
+	EnableGRPCGateway bool `json:"enable-grpc-gateway"`
+
 	// TO BE DEPRECATED
 
 	// LogPkgLevels is being deprecated in v3.5.
```

Comment:
what about just `--grpc-gateway`?

Reply:
I think `enable-grpc-gateway` make it clearer?

Reply:
enable-grpc-gateway is fine with me.

> I think `enable-grpc-gateway` make it clearer?

Yeah this is fine with me actually current `--gateway` flag may lead user to wrong assumption. `enable` is better solution.

Reply:
Then I'll keep the name